buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.java

```diff
@@ -141,7 +141,7 @@ private static void setupApiInformationAnalysis(Project project, LibraryExtensio
                   task.setSourceSet(mainSourceSet);
                   task.setBaselineFile(project.file("baseline.txt"));
                   if (project.hasProperty("updateBaseline")) {
```

Comment:
My mistake, I cannot suggest multiline changes in Github. With this change, you don't need the enclosing if/else.

Reply:
done